frontend/sw360-portlet/src/main/resources/META-INF/resources/html/licenses/includes/detailTodos.jspf

```diff
@@ -95,15 +95,15 @@
                     <td><sw360:DisplayBoolean value="${oblig.development}" defined="${oblig.setDevelopment}"/></td>
                     <td><sw360:DisplayBoolean value="${oblig.distribution}" defined="${oblig.setDistribution}"/></td>
                     <td><sw360:DisplayMap value="${oblig.customPropertyToValue}"/></td>
-                    <core_rt:if test="${oblig.listOfobligationSize > 0}">
-                        <td>
+                    <td>
+                        <core_rt:if test="${oblig.listOfobligationSize > 0}">
                             <ul>
                                 <core_rt:forEach var="ob" varStatus="status" items="${oblig.obligations}">
```

Comment:
It should be  `${oblig.listOfobligation}` instead of `${oblig.obligations}`

Reply:
Changes made